src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: $main-width;
```

Comment:
The card width is set to a fixed value. This can cause issues with responsiveness. Consider using a relative unit like percentages or use media queries to adjust the width based on the viewport size.